srcpkgs/Waybar/template

```diff
@@ -31,7 +31,7 @@ changelog="https://github.com/Alexays/Waybar/releases"
 distfiles="https://github.com/Alexays/Waybar/archive/refs/tags/${version}.tar.gz
  https://github.com/HowardHinnant/date/archive/refs/tags/v${_date_version}.tar.gz
  https://github.com/mesonbuild/hinnant-date/archive/refs/tags/${_mesonbuild_date_ver}.tar.gz"
```

Comment:
this looks like an ugly hack, is it still required?

Reply:
We not have chrono-date packaged yet, and this is necessary for clock/date module to work.

Reply:
doesn't look too complex to package, the template will probably be straightforward

Reply:
https://github.com/void-linux/void-packages/pull/41750 
if everything is fine with this PR and it is merged, the next step then is to change the Waybar template so that it uses the chrono-date package and no longer this 'ugly hack'.

Reply:
Any update on this?

Reply:
https://github.com/void-linux/void-packages/commit/cefab22ecb99aa8e6dbb8b8f234da96de7738cc1 with that the 'ugly hack' has resolved.